src/main/java/io/logz/jmx2logzio/objects/Metric.java

```diff
@@ -6,10 +6,10 @@
 import java.time.Instant;
 import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.stream.Collectors;
 
 public class Metric {
```

Comment:
@yyyogev Do you think the number of parameters can grow in the future?

Reply:
Not sure I follow